packages/adp-tooling/src/writer/options.ts

```diff
@@ -178,21 +182,35 @@ function addOpenSourceMiddlewares(ui5Config: UI5Config, config: AdpWriterConfig)
  * @returns list of required tasks.
  */
 function getAdpCloudCustomTasks(config: AdpWriterConfig & { target: AbapTarget } & { app: CloudApp }): CustomTask[] {
+    let target: CloudCustomTaskConfigTarget;
+    if (config.target.destination) {
```

Comment:
Maybe it's good to have "?." check just in case?
```suggestion
    if (config?.target?.destination) {
```

Reply:
Fixed in [857a668](https://github.com/SAP/open-ux-tools/pull/2362/commits/857a6683440c925497339881d418afaa488ef86c)